lib/plain/endpoints/team-space-membership.ts

```diff
@@ -0,0 +1,86 @@
+import { AxiosInstance } from 'axios'
+import cloneDeep from 'lodash/cloneDeep'
+import * as raw from './raw'
+import { GetSpaceParams, QueryParams, CollectionProp, GetOrganizationParams } from './common-types'
+import {
+  TeamSpaceMembershipProps,
+  CreateTeamSpaceMembershipProps,
+} from '../../entities/team-space-membership'
+
+type GetTeamSpaceMembershipParams = GetSpaceParams & { teamSpaceMembershipId: string }
+
+const getBaseUrl = (params: GetSpaceParams) => `/spaces/${params.spaceId}/team_space_memberships`
+
+const getEntityUrl = (params: GetTeamSpaceMembershipParams) =>
+  `${getBaseUrl(params)}/${params.teamSpaceMembershipId}`
+
+export const get = (http: AxiosInstance, params: GetTeamSpaceMembershipParams) =>
+  raw.get<TeamSpaceMembershipProps>(http, getEntityUrl(params))
+
+export const getMany = (http: AxiosInstance, params: GetSpaceParams & QueryParams) =>
+  raw.get<CollectionProp<TeamSpaceMembershipProps>>(http, getBaseUrl(params), {
+    params: params.query,
+  })
+
+export const getForOrganization = (
+  http: AxiosInstance,
+  params: GetOrganizationParams & { teamSpaceMembershipId: string }
+) => {
+  return raw.get<TeamSpaceMembershipProps>(
+    http,
+    `/organizations/${params.organizationId}/team_space_memberships/${params.teamSpaceMembershipId}`
+  )
+}
+
+export const getManyForOrganization = (
+  http: AxiosInstance,
+  params: GetOrganizationParams & QueryParams & { teamId?: string }
+) => {
+  const query = params.query || {}
+  if (params.teamId) {
```

Comment:
I'd rather leave this magic in the organization api of the old client.

Reply:
I wouldn't say it's magic but rather a part of the how API works. We didn't leave any implementation details in the official client in other methods, so I'd rather be consistent here.

Reply:
I think it's something different to what we provide in other places. But no strong opinion on this one.